hbase-server/src/main/java/org/apache/hadoop/hbase/master/ActiveMasterManager.java

```diff
@@ -178,10 +214,14 @@ boolean blockUntilBecomingActiveMaster(
           // We are the master, return
           startupStatus.setStatus("Successfully registered as active master.");
           this.clusterHasActiveMaster.set(true);
+          activeMasterServerName.set(sn);
           LOG.info("Registered as active master=" + this.sn);
           return true;
         }
 
+        // Invalidate the active master name so that subsequent requests do not get any stale
+        // master information. Will ne re-fetched if needed.
```

Comment:
nit: will **be** re-fetched

Reply:
will fix this in next patch set, thanks for catching.